github-extra/resources/gitconfig

```diff
@@ -32,10 +32,7 @@
 	format = html
 
 [http]
-	sslcainfo = /bin/curl-ca-bundle-ghfw.crt
-
-[sendemail]
-	smtpserver = /bin/msmtp.exe
+	sslcainfo = /ming32/usr/ssl/ca-bundle-ghfw.crt
```

Comment:
So I fat-fingered this and got a lovely message once I was about to update the submodule reference:

```
$ git fetch
fatal: unable to access 'https://github.com/github/ghfw-build-extra.git/': error setting certificate verify locations:
  CAfile: C:\git-sdk-32\mingw32/ming32/usr/ssl/ca-bundle-ghfw.crt
  CApath: none
```

However the duplication of the `mingw32` entries here makes me feel like we've missed something (and this shouldn't have the prefix when defined in the config).